pkg/test/framework/components/policybackend/policybackend.go

```diff
@@ -397,7 +406,7 @@ func (p *policyBackend) CreateConfigSnippet(name string) string {
 kind: bypass
 metadata:
   name: %s
-  namespace: istio-system
+  namespace: {{.TestNamespace}}
```

Comment:
Err ... TestNamespace? I assume that's so it will work with BookInfo? The apps are deployed to the DependencyNamespace ... will the policy backend work in that case?